temporal-opentracing/src/main/java/io/temporal/opentracing/SpanCreationContext.java

```diff
@@ -0,0 +1,128 @@
+/*
+ *  Copyright (C) 2020 Temporal Technologies, Inc. All Rights Reserved.
+ *
+ *  Copyright 2012-2016 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *  Modifications copyright (C) 2017 Uber Technologies, Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"). You may not
+ *  use this file except in compliance with the License. A copy of the License is
+ *  located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ *  or in the "license" file accompanying this file. This file is distributed on
+ *  an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *  express or implied. See the License for the specific language governing
+ *  permissions and limitations under the License.
+ */
+
+package io.temporal.opentracing;
+
+/**
+ * Used when creating an OpenTracing span and provides contextual information used for naming and
+ * tagging OT spans.
+ */
+public class SpanCreationContext {
+
+  private SpanOperationType spanOperationType;
+  private String operationName;
```

Comment:
I think `operationName` should be renamed to something like `actionName`, `resourceName`, or AT LEAST `opName`.
The reason is that `operationName` is already used/reserved by both OpenTracing and OpenTelemetry and it means "span name". Reusing it here looks very ambiguous.